test/main.js

```diff
@@ -484,5 +484,49 @@ describe('glob-stream', function() {
       gs.create.bind(gs, ['!a', '!b'], {cwd: __dirname}).should.throw(/Missing positive glob/);
     });
 
+    it('should emit error on singular glob when file not found', function(done) {
+      var stream = gs.create('notfound');
+      should.exist(stream);
+      stream.on('error', function(err) {
+        err.should.match(/File not found with singular glob/);
+        done();
+      });
+    });
+
+    it('should not emit error when multiple globs not found', function(done) {
+      var stream = gs.create(['notfound', 'alsonotfound']);
+      should.exist(stream);
+      stream.on('error', function() {
+        should.fail();
+      });
+
+      // For some reason `end` isn't called unless `data` is first?
+      stream.on('data', function(){});
+      stream.once('end', done);
+    });
+
+    it('should not emit error on glob containing {} when not found', function(done) {
+      var stream = gs.create('notfound{a,b}');
+      should.exist(stream);
+      stream.on('error', function() {
+        should.fail();
```

Comment:
I'm going with `throw new Error('Error was emitted');`, because otherwise it just complains that `done()` is being called twice. I figure I could also just use `_.once(done)`, but I don't see any advantage over this.